src/helpers/color.ts

```diff
@@ -130,7 +130,10 @@ function rgbaStringToHex(color: Color): Color {
     throw new Error("invalid color");
   } else if (stringVals.length === 4) {
     const alpha = parseFloat(stringVals.pop() || "1");
-    alphaHex = Math.round((alpha || 1) * 255);
+    if (isNaN(alpha)) {
```

Comment:
could we also add a check (or automatically clipping) that the value of alpha is between 0 and 1 ?

Reply:
`toHex` (that call this function) already checks that the resulting color is valid. I added some tests to make sure it works :)

Reply:
thanks :) I didn't check that far :'(